### PR TITLE
Bug/fetch img description

### PR DESCRIPTION
**Pull request by KShih**
This PR adds functionality in `refresh_apartment`:
- crawl image from the URL
- crawl description from the URL

And also fix the code of display image in `search.html` and `apartment.html`.

image
image

### CODE REVIEW

**Comment by ab7289** [comment]
would it be possible to get some more testing on this? We're losing almost an entire percentage point in this PR.

**Comment by KShih** [comment]
@ab7289 
Do these ten data are created before?
Maybe try to clear your old data.

**Comment by ab7289** [comment]
> @ab7289
> Do these ten data are created before?
> Maybe try to clear your old data.

This was with a completely new database. I deleted `db.sqllite3` prior to checking out the branch and re-ran all migrations.

**Comment by KShih** [comment]
> > @ab7289
> > Do these ten data are created before?
> > Maybe try to clear your old data.
> 
> This was with a completely new database. I deleted `db.sqllite3` prior to checking out the branch and re-ran all migrations.

Could you check out the /admin/ ?

**Comment by ab7289** [comment]
> > > @ab7289
> > > Do these ten data are created before?
> > > Maybe try to clear your old data.
> > 
> > 
> > This was with a completely new database. I deleted `db.sqllite3` prior to checking out the branch and re-ran all migrations.
> 
> Could you check out the /admin/ ?

Yeah, no Images or descriptions. Maybe someone else can try to see if it was just a fluke on my end?

**Comment by ab7289** [comment]
I got this working locally. I think the issue was that the celery worker was either running from a different branch due to having multiple terminal windows open.

File: refresh_apartment/tests.py [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
 import os
 
-# Create your tests here.
+NORMAL_URL = "craigslist-url-normal.html"
+NO_PIC_URL = "craigslist-url-no-pic.html"
+NO_DESCRIPTION_URL = "craigslist-url-no-des.html"
+NO_PIC_NO_DESCRIPTION_URL = "craigslist-url-no-pic-no-des.html"
+
+
+class MockRequestResponse:
```
Comment (ab7289) [inline refresh_apartment/tests.py]: I like it 👍

**Comment by ab7289** [comment]
Looks awesome! Did a test refresh and pictures and descriptions were loading and the updated test coverage is great.

If you could just add one last (small) test for line 185 I think we're good to go.

Screen Shot 2019-12-03 at 11 20 31 PM

**Comment by KShih** [comment]
> Looks awesome! Did a test refresh and pictures and descriptions were loading and the updated test coverage is great.
> 
> If you could just add one last (small) test for line 185 I think we're good to go.
> 
> Screen Shot 2019-12-03 at 11 20 31 PM

Good catch! Updated.

**Pull request merged**